Expose DWExportHeader API function

fleimgruber opened the pull request:
Minimalistic wrapper around DWExportHeader from the DWDataReader
API. The file name "local.xml" was chosen since it mirrors the name
DEWESoft gives the XML file when unzipping a d7s or dxs setup file.

* `dwdatareader/__init__.py` (DWFile.export_header): Wrap DWExportHeader
  API function.
* `tests.py` (test_export_header): Assert that the XML file has been
  exported with the conventional name.

fleimgruber commented:
After some feedback I got for this, it would probably be better to have the file name as an argument to the wrapper function as this resembles most closely the C API.

costerwi commented:
Yes, having the file name as an argument sounds good. You should be able to push that to your existing branch.

fleimgruber commented:
Updated for file name and fixed test. If you approve of this, I will squash the 3 commits to a single one before merging, makes the history smoother I think. Or do you squash anyways when merging?

costerwi commented:
There is a button for "squash and merge" which seemed to work.

Thanks!
